lib/services/bitcoind.js

```diff
@@ -1083,9 +1085,10 @@ Bitcoin.prototype.getAddressBalance = function(addressArg, options, callback) {
 Bitcoin.prototype.getAddressUnspentOutputs = function(addressArg, options, callback) {
   var self = this;
   var queryMempool = _.isUndefined(options.queryMempool) ? true : options.queryMempool;
+  var showImmatureBTs = options.showImmatureBTs == 1 ? true : false;
   var addresses = self._normalizeAddressArg(addressArg);
   var cacheKey = addresses.join('');
-  var utxos = self.utxosCache.get(cacheKey);
+  var utxos = self.utxosCache.get(cacheKey+showImmatureBTs);
```

Comment:
The cache key seems to be a normalized version of the address.  And if showImmatureBTs is true, the cache key will have a "1" at the end of it.  Is that correct?  I believe that bitcoin-based addresses are variable-length from 26-35 alphanumeric characters.  It seems unlikely, but are there concerns of a collision?  If so, we could just add a fixed string (such as "ShowImmature") instead of 1.